lib/mayaUsd/ufe/MayaUIInfoHandler.cpp

```diff
@@ -92,12 +92,7 @@ bool isOrphaned(const Ufe::SceneItem::Ptr& mayaItem)
 namespace MAYAUSD_NS_DEF {
 namespace ufe {
 
-MayaUIInfoHandler::MayaUIInfoHandler()
-    : Ufe::UIInfoHandler()
-{
-}
```

Comment:
Empty constructors that are just really "default" were replaced (in .h) with default.